static/js/project/controllers/my-project.controller.js

```diff
@@ -59,6 +59,7 @@
         }
 
         function createProject(){
+            Project.clean();
```

Comment:
cleans project on Create in case user navigates away from the task creation wizard